### PR TITLE
Arachne 1852 front end for arachne global search

### PR DESCRIPTION
**Pull request by johnSamilin**

### CODE REVIEW

File: src/modules/Portal/components/Search/NavbarInput/container.js [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+      fetchResults: ducks.actions.search.query,
+    };
+  }
+
+  mergeProps(stateProps, dispatchProps, ownProps) {
+    return {
+      ...ownProps,
+      ...stateProps,
+      ...dispatchProps,
+      onSubmit() {
```
Comment (pavgra) [inline src/modules/Portal/components/Search/NavbarInput/container.js]: ?

File: src/modules/Admin/components/SystemSettings/presenter.jsx [hunk 2]
```diff
@@ -86,14 +86,30 @@ function SystemSettings(props) {
                 label="Reindex Expert Finder"
                 onClick={() => solrReindex({ domain: 'users' })}
               />
-            </div>,
+            	</div>,
```
Comment (pavgra) [inline src/modules/Admin/components/SystemSettings/presenter.jsx]: ouch

File: src/modules/Portal/components/Search/Results/presenter.jsx [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+    searchParams,
+    isLoading,
+    paginationDetails,
+    searchQueryDecode,
+    searchQueryEncode,
+    filterFields = [],
+  } = props;
+
+  const filter = cloneDeep(filterFields);
+  filter[0].options = filter[0].options.map((field) => ({
```
Comment (pavgra) [inline src/modules/Portal/components/Search/Results/presenter.jsx]: What is this?

File: src/modules/Portal/components/Search/Results/List/Crumb/index.js [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+ * limitations under the License.
+ *
+ * Company: Odysseus Data Services, Inc.
+ * Product Owner/Architecture: Gregory Klebanov
+ * Authors: Pavel Grafkin, Alexander Saltykov, Vitaly Koulakov, Anton Gackovka, Alexandr Ryabokon, Mikhail Mironov
+ * Created: january 23, 2018
+ *
+ */
+
+import Crumb from './presenter';
```
Comment (pavgra) [inline src/modules/Portal/components/Search/Results/List/Crumb/index.js]: Let's unify Breadcrumbs across system

File: src/modules/Portal/components/Search/NavbarInput/container.js (hunk 1 above)
Comment (johnSamilin) [reply src/modules/Portal/components/Search/NavbarInput/container.js]: form component should have that handler. it's generic

**Pull request merged**